packages/gatsby-source-contentful/README.md

```diff
@@ -144,6 +144,12 @@ Downloads and caches `ContentfulAsset`'s to the local filesystem. Allows you to
 
 You can pass in any other options available in the [contentful.js SDK](https://github.com/contentful/contentful.js#configuration).
 
+**`localeFilter`** [function][optional] [default: `() => true`]
+
+Possibility to limit how many locales/nodes are created in graphQL. This can limit the memory usage by reducing the amout of nodes created. Useful if you have a large space in contentful and only want to get the data from one selected locale.
+
+For example, to filter locales on only germany `localeFilter: locale => locale.code === 'de-DE'`
```

Comment:
I just tried it with space we are using in our example, and was confused why this wasn't working, but apparently code for German localization in that space is just `de`. I think I'll add some console output that lists locale so it's easier for users to specify locale code

Reply:
Ok, output is getting messy if I do it
![Screenshot 2019-04-16 at 12 32 04](https://user-images.githubusercontent.com/419821/56202570-a93e0480-6043-11e9-94bd-e05b99d046ad.png)

So maybe instead of that (at least before https://github.com/gatsbyjs/gatsby/pull/13089 land) We could do:
```suggestion
For example, to filter locales on only germany `localeFilter: locale => locale.code === 'de-DE'`

List of locales and their codes can be found in Contentful app -> Settings -> Locales
```

Reply:
That sounds totally fine to me! I actually also got confused when 

(I can't push commits to Robin's PR though I realized, so it's either I fork that PR and create my own with that change, or if you have the possibility to add commits, feel free to add it)

Reply:
I'll try to push it directly and merge it.

I checked - in locale settings you can pick both `de` and `de-DE` there - so it depends on which one you chose from dropdown it will use that code. But locale overview also display codes so that should be fine to add here.